tests/upgrades/test_usergroup.py

```diff
@@ -89,15 +108,21 @@ def test_post_verify_usergroup_membership(self, request, dependent_scenario_name
         :expectedresults: After upgrade, user group membership should remain the same and LDAP
         auth update should work.
```

Comment:
TBH, this line is not covered by this scenario of updating the user etc is not captured! 

I would either leave it @omkarkhatavkar to handle in this refactor or Teams should handle it as part of coverage improvement plan.